ui/app/AppLayouts/Chat/popups/GroupInfoPopup.qml

```diff
@@ -35,13 +35,14 @@ StatusModal {
     property bool isAdmin: false
     property Component pinnedMessagesPopupComponent
 
+    property var chatContentModule
+
     function resetSelectedMembers(){
         pubKeys = []
 
-        memberCount = channel ? channel.members.rowCount() : 0
+        memberCount = popup.chatContentModule.usersModule.model.rowCount()
```

Comment:
@borismelnik I see this is merged into `base_bc`, but in the comment I added [here](https://github.com/status-im/status-desktop/pull/4583#pullrequestreview-863245938) I deliberately used `count` instead of `rowCount()`, since `rowCount()` is not a slot of that model and I am wondering if that works?